src/TiledArray/dist_array.h

```diff
@@ -227,6 +228,59 @@ class DistArray : public madness::archive::ParallelSerializableObject {
                 std::shared_ptr<pmap_interface>())
       : pimpl_(init(world, trange, shape, pmap)) {}
 
+  template<typename T>
+  DistArray(World& world, std::initializer_list<T> il) :
+    DistArray(array_from_il<DistArray_>(world, il)) {}
+
+  template<typename T>
+  DistArray(World& world,
+            std::initializer_list<std::initializer_list<T>> il) :
+    DistArray(array_from_il<DistArray_>(world, il)) {}
+
+  template<typename T>
+  DistArray(World& world,
+            std::initializer_list<
```

Comment:
does not appear to be formatted ... I don't have auto-formatting workflow since it creates too much friction. there is a bundled .clang-format , and (best) use pre-commit hook ... the docs have not been updated with pre-commit info, so see the analogous info for MPQC: https://valeevgroup.github.io/mpqc-docs/v4-private/contrib-workflow.html